### ISSUE TITLE
Problems on new device

### ISSUE THREAD

**Issue by COMMANDERJONES** [radimitrov/CSharpShellApp#92]
I have been using this app on my old phone (Samsung galaxy j7 2016) with no problems.
Now my device is the Samsung galaxy m51 and i have the following 2 problems:
1- If i exit the app without closing it, for example pressing the home button then opening it back, it freezes and i have to exit the application and reopen it.
2- Sometimes, after opening the app, the file that i had open before gets copied to the file containing the main function.

I dont know if my device is causing these problems, or perhaps a new update. If it is a device issue, please let me know. Thanks.

**Comment by thong927** [comment]
<a href ="https://github.com/radimitrov/CSharpShellApp/issues/89 ">Is the second error that you describe the same as this one? </a>

**Comment by COMMANDERJONES** [comment]
Hello,

If i understand correctly what the other person you mentioned described, no.
In my case it doesn't happen while i am writing the code, nor does a
rectangular box appear.
It happens sometimes when, while viewing a file, you freeze the app (as
mentioned before), then re-opening the application.

Στις Τετ, 28 Απρ 2021, 09:46 ο χρήστης thong927 ***@***.***>
έγραψε:

**Comment by radimitrov** [comment]
Yes, the second issue is the mentioned one. The mentioned fix there will work once I finally manage to finish the big update.
And the sometimes is part is definetelly correct. That is issue is too unpredictable in frequency and location. 

The freezing problem feels strange... Has to be something with states or activities, but not encountered on most devices, it seems.
Though it might just be "auto-solved" already since I'm not keeping most of that original code for the update.

**Comment by COMMANDERJONES** [comment]
Maybe the freezing has to do with my device trying to kill the app in the
background? I never had the freezing in my old device. Anyways, thank you
very much for the feedback, i am waiting for the update and hope it fixes
everything. Cheers.

Στις Τετ, 28 Απρ 2021, 17:38 ο χρήστης Radostin Dimitrov <
***@***.***> έγραψε:
